docs/source/app-dev/authorization.rst

```diff
@@ -7,124 +7,189 @@ Authorization
 #############
 
 When developing Daml applications using SDK tools,
-your local setup will most likely not verify any authorization -
+your local setup will most likely not perform any ledger API request authorization --
 by default, any valid ledger API request will be accepted by the sandbox.
 
-To run your application against a :doc:`deployed ledger </deploy/index>`, you will need to add authorization.
+This is not the case for participant nodes of :doc:`deployed ledgers </deploy/index>`.
+They check for every ledger API request whether the request contains an access token that is valid and sufficient to authorize the request.
+You thus need to add support for authorization using access token to your application to run it against a deployed ledger.
 
 Introduction
 ************
 
-The :doc:`Ledger API </app-dev/ledger-api>` is used to request changes to the ledger (e.g., "*Alice
-wants to exercise choice X on contract Y*"), or to read data from the ledger (e.g., "*Alice wants to
-see all active contracts*").
+Your Daml application sends requests to the :doc:`Ledger API </app-dev/ledger-api>` exposed by a participant node to submit changes to the ledger
+(e.g., "*exercise choice X on contract Y as party Alice*"), or to read data from the ledger
+(e.g., "*read all active contracts visible to party Alice*").
+Your application might send these requests via a middleware like the :doc:`JSON API </json-api/index>`.
 
-What requests are valid is defined by :ref:`integrity <da-model-integrity>` and :ref:`privacy <da-model-privacy>` parts the :ref:`Daml Ledger Model <da-ledgers>`.
-This model is defined in terms of :ref:`Daml parties <glossary-party>`,
-and does not require any cryptographic information to be sent along with requests.
+Whether a participant node *can* serve such a request depends on whether the participant node hosts the respective parties, and
+whether the request is valid according to the :ref:`Daml Ledger Model <da-ledgers>`.
+Whether a participant node *will* serve such a request to a Daml application depends on whether the application accompanied the
```

Comment:
A little loquacious - might be better as "whether the request includes an access token" or something similar.

Reply:
thx. addressed.